aspnetcore/fundamentals/index.md

```diff
@@ -68,61 +67,45 @@ An ASP.NET Core app builds a *host* on startup. The host is an object that encap
 
 The main reason for including all of the app's interdependent resources in one object is lifetime management: control over app startup and graceful shutdown.
 
-The code to create a host is in `Program.Main` and follows the [builder pattern](https://wikipedia.org/wiki/Builder_pattern). Methods are called to configure each resource that is part of the host. A builder method is called to pull it all together and instantiate the host object.
-
 ::: moniker range=">= aspnetcore-3.0"
 
-`CreateHostBuilder` is special name that identifies the builder method to external components, such as [Entity Framework](/ef/core/).
+Two host versions are available: Generic Host and Web Host. In ASP.NET Core 3.0 and later, Generic Host is recommended, and Web Host is available only for backwards compatibility.
```

Comment:
Since this is versioned content, it's probably ok to just state it ...

```suggestion
Two hosts are available: Generic Host and Web Host. The Generic Host is recommended, and the Web Host is available for backwards compatibility.
```